<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startServer">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>430</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="networkLatency">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="cpuLatency">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="mainActiveRooms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Active Rooms:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainConnectedClients">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connected Clients:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Network Latency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU Latency</string>
    </property>
   </widget>
   <widget class="QLabel" name="nw1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nw9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu1">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>290</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu8">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu6">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu7">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu9">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cpu3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>230</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mainPacketLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Packets/S:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
